Server/Server.Core/Server.Core.Static/Server.cpp

```diff
@@ -1,13 +1,78 @@
 #include <future>
 
 #include "Server.hpp"
-#include "DataAccess.Postgre/PostgreRepositoryManager.hpp"
+
 
 namespace Server
 {
 using namespace DataAccess;
 using UtilityTime::safe_localtime;
 
+
+
+Server::Server() { /* do nothing*/ }
```

Comment:
If we want to stick to the logic of the builder server, it must have default values passed to the default constructor